generic/tests/cfg/kdump.cfg

```
@@ -16,6 +16,8 @@
     # crash_cmd = "echo c > /proc/sysrq-trigger"
     # If you want configure multiline, split lines with ";", like:
     # kdump_config = configure line1; config line2
+    kernel_param_cmd = "grubby --update-kernel=`grubby --default-kernel` --args=crashkernel=1G-4G:192M,4G-64G:256M,64G-:512M"
```

@Kemama I think @PaulYuuu means the default are different so the args should be different for arm, so we might want:
```suggestion
default_crashkernel = 1G-4G:192M,4G-64G:256M,64G-:512M
aarch64:
    default_crashkernel = 1G-4G:256M,4G-64G:320M,64G-:576M
kernel_param_cmd = "grubby --update-kernel=`grubby --default-kernel` --args=crashkernel=${default_crashkernel}"
```
And:
```
def_kernel_param_cmd = ("grubby --update-kernel=`grubby --default-kernel` --args=crashkernel=%s" % params['default_crashkernel'])
```